server/routers/viewer.tsx

```diff
@@ -295,6 +295,7 @@ const loggedInViewerRouter = createProtectedRouter()
             AND: [
               { NOT: { status: { equals: BookingStatus.CANCELLED } } },
               { NOT: { status: { equals: BookingStatus.REJECTED } } },
+              { NOT: { paid: { equals: false } } }, // hide unpaid bookings https://app.clickup.com/t/27pjd5c
```

Comment:
@zach-at-the-skills what about `{ NOT: { confirmed: { equals: false } } } `

Reply:
Seems equivalent, but maybe that will be better for the future?